plugins/discord/back.js

```diff
@@ -4,8 +4,8 @@ const { dialog, app } = require("electron");
 
 const registerCallback = require("../../providers/song-info");
 
-// Application ID registered by @semvis123
-const clientId = "790655993809338398";
+// Application ID registered by @xn-oah
+const clientId = "937234785716285471";
```

Comment:
Does that need to be updated/can it change something for people relying on the plugin? If it's not necessary (which seems to be the case), better not updating it - a good alternative if you want to use another app ID is to make it a parameter in the plugin config, so that everyone is free to choose what they want to use :)

Reply:
to use  a custom icon on discord presence - you need to register an app and upload assets
(see https://discord.com/developers/docs/topics/gateway#activity-object-activity-asset-image)

he uploaded a new icon (paused icon) to the app's assets on discord

`activityInfo.smallImageKey = "paused";` is the key of the paused icon he uploaded to the dashboard